pkg/cluster/nodes/node.go

```diff
@@ -102,7 +102,7 @@ func (n *Node) CopyFrom(source, dest string) error {
 // WaitForDocker waits for Docker to be ready on the node
 // it returns true on success, and false on a timeout
 func (n *Node) WaitForDocker(until time.Time) bool {
-	return tryUntil(until, func() bool {
+	return tryUntil(until, time.Second, func() bool {
```

Comment:
hmm, how did we pick these values?

the downside to this is increasing the latency from ready -> cluster up progressing
these commands are somewhat self throttling by taking time themselves.

Reply:
> hmm, how did we pick these values?

experiment from yesterday.

> the downside to this is increasing the latency from ready -> cluster up progressing
these commands are somewhat self throttling by taking time themselves.

it executes something like a 100 kubectl calls until it gets the signal that the CP is ready.

Reply:
> it executes something like a 100 kubectl calls until it gets the signal that the CP is ready.

sounds working as intended 😉 

the docker call should be much more throttled, this one is much slower

Reply:
SGTM.  :+1  Do we need to do more experiments on different performance machines? I have some local machines available for testing.